components/director/internal/repo/get.go

```diff
@@ -67,14 +67,16 @@ func (g *universalSingleGetter) unsafeGet(ctx context.Context, conditions Condit
 		return err
 	}
 
+	log.Debug("Building DB query...")
 	query, args, err := buildSelectQuery(g.tableName, g.selectedColumns, conditions, orderByParams)
 	if err != nil {
+		log.Errorf("Error while building DB query: %s", query)
```

Comment:
Is this log needed isn't this logged in the outher scope?
If not add the error to the log also.

Reply:
I've removed the log because the error is propagated to the outer scopes